code/_globalvars/misc.dm

```diff
@@ -78,4 +78,9 @@ GLOBAL_VAR_INIT(timezoneOffset, 0) // The difference betwen midnight (of the hos
 // However it'd be ok to use for accessing attack logs and such too, which are even laggier.
 var/fileaccess_timer = 0
 
-GLOBAL_VAR_INIT(gametime_offset, 432000) // 12:00 in seconds
\ No newline at end of file
+GLOBAL_VAR_INIT(gametime_offset, 432000) // 12:00 in seconds
+
+//printers shutdown if too much shit printed
+var/items_printed = 0
+var/max_items_printed = 300
```

Comment:
300 seems a little high. I'd be tempted to go with 100 or less.
I'd also be tempted to rename both of these vars 'copier_items_printed' and 'copier_max_items', because 'items_printed' is generic enough that it might conflict with a local var on some other object in future.